multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/metadata/BlueGreenDeployMetadata.java

```diff
@@ -92,6 +87,26 @@ public static OperationMetadata getMetadata() {
                                                                                  .id(Variables.RESOURCES_FOR_DEPLOYMENT.getName())
                                                                                  .type(ParameterType.STRING)
                                                                                  .build())
+                                         .addParameter(ImmutableParameterMetadata.builder()
+                                                                                 .id(Variables.APPS_START_TIMEOUT_COMMAND_LINE_LEVEL.getName())
```

Comment:
Rename here and everywhere the term "COMMAND_LINE_LEVEL" with "Process variable".
Make sure that everywhere is replaced, including method names, local variables and constants.

This is needed because from the perspective of the backend multiapps-controller treat all clients equally without differentiating command line or others.
Also we are currently name these parameters as process variables .